docs-website/pages/app.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
I'd say it's a lot nicer to put the documentation in the `docs` folder and have the `docs-website` be the _code_ of the website, without actual content.

Reply:
I see your point. I'll take another look at the file organisation once I've got the content in place